references/detection/transforms.py

```diff
@@ -326,3 +326,78 @@ def forward(
                 )
 
         return image, target
+
+
+class FixedSizeCrop(nn.Module):
+    def __init__(self, size, fill=0, padding_mode="constant"):
+        super().__init__()
+        size = tuple(T._setup_size(size, error_msg="Please provide only two dimensions (h, w) for size."))
+        self.crop_height = size[0]
+        self.crop_width = size[1]
+        self.fill = tuple(int(x) for x in fill)  # TODO: Fill is currently respected only on PIL. Apply tensor patch.
```

Comment:
fill by default is int and `for x in fill` will fail with default value

Reply:
This quick and dirty fix was necessary for PIL images because there is a bug on `pad()`. As we discussed on #5568 these patches needed to be fixed on a new PR. I'm currently busy but I'll patch it soon.

Reply:
The comment is about : 
```
t = FixedSizeCrop(size=(36, 14))

> self.fill = tuple(int(x) for x in fill)  # TODO: Fill is currently respected only on PIL. Apply tensor patch.
> 
> TypeError: 'int' object is not iterable
```

Reply:
Oops I see what you say. Let me patch now on main then and avoid this. Thanks for flagging it.